docs/src/features/settings.md

```diff
@@ -9,16 +9,17 @@
     <!--settings-start-->
 
     ```python
-    # If "idom" cache is not configured, then we will use "default" instead
+    # If "idom" cache is not configured, then "default" will be used
+    # IDOM expects a multiprocessing-safe and thread-safe cache backend.
     CACHES = {
     "idom": {"BACKEND": ...},
     }
 
-    # Maximum seconds between two reconnection attempts that would cause the client give up.
-    # 0 will disable reconnection.
-    IDOM_WS_MAX_RECONNECT_TIMEOUT = 604800
+    # Maximum seconds between reconnection attempts before giving up.
+    # Use `0` to prevent component reconnection.
+    IDOM_RECONNECT_MAX = 259200
```

Comment:
To clarify, IDOM never actually give up trying to reconnect at the moment. The max reconnect is the max time between attempts.

Reply:
It's a bit of a preemptive rename and description change.

See https://github.com/idom-team/django-idom/pull/120#issuecomment-1378170208